### ISSUE TITLE
Gutter Not Showing Colors

### ISSUE THREAD

**Issue by venatiodecorus** [facelessuser/BracketHighlighter#284]
Hi,

I've been trying to get colors to display properly for the icons in the gutter. They don't seem to work for me with any theme or any color scheme. They're always just white icons. I tried using the Aprosopo theme mentioned in the docs as well, and no luck with that. Any ideas on why the icons might not be displaying colors? I'm running ST3 on Arch Linux.

**Comment by facelessuser** [comment]
With so little info from you I have no idea what you've done or what you haven't.  You would need to post your personal `bh_core.sublime-settings`, your `Preferences.sublime-settings` (so I can verify you are using the color scheme you say you are using), and your color scheme.  It sounds like you have at least read of the docs some, so that is good.

OS should not matter.

If you can provide more info that would be great.  Make sure to also look in your console to see if there are errors; people sometimes mess up there settings files so things aren't getting parsed right.

**Comment by venatiodecorus** [comment]
Hello, thank you for responding so quickly. My `bh_core.sublime-settings` file is empty. Here is my preferences file as it currently stands: https://gist.github.com/venatiodecorus/c794bafce82ec653a6f9

I checked my console and I don't see any errors. My current color scheme is the one I've been using but I did try a bunch of different themes to see if I could get the colors to work.

**Comment by facelessuser** [comment]
Please read http://facelessuser.github.io/BracketHighlighter/customize/#my-personal-configurations and note my theme **and** my settings.  Your settings file must point to colored scores in your theme.  I provide mind as an example.  You can try out my setup, and use that as a guide.

**Comment by facelessuser** [comment]
I am assuming you got your answer?

**Comment by venatiodecorus** [comment]
Not really. I guess I'd have to go through and set colors for each bracket type manually. I haven't had time to make that effort for such a small payoff. It was nice when the colors just seemed to work though. But for now, single color gutter brackets work fine for me.

**Comment by facelessuser** [comment]
I don't know which ones support colors.  The color scopes in my example.  They don't tell me anything.  I know mine do, but since I use different colors for each bracket, you have to modify your settings file to use theme.  If you want it to work with your current color scheme, you just need to point the settings at scopes that exist in your scheme instead of what I show in the example.

**Comment by facelessuser** [comment]
Tell you what.  Just do it the old way as it talks about in the settings:

```js
        "default": {
            "icon": "dot",
            // BH1's original default color for reference
            // "color": "entity.name.class",
            "color": "brackethighlighter.default",
            "style": "underline"
        },
```

Just change `"color": "brackethighlighter.default"` to `"color": "entity.name.class"` and it will pick up whatever color in you scheme that is assigned to `entity.name.class`.  Or change it to something else like `comment` or `string`.  You don't have to individually configure them; I do, but if you just tweak the default only, **all** brackets will use the default color.
